Data_Structures_Questions.md

```diff
@@ -2,19 +2,19 @@ Answer the following questions for each of the data structures you implemented a
 
 ## Queue
 
-1. What is the runtime complexity of `enqueue`?
+1. What is the runtime complexity of `enqueue`? O(n)
 
-2. What is the runtime complexity of `dequeue`?
+2. What is the runtime complexity of `dequeue`? O(n)
```

Comment:
If the remove from head method that is used within the dequeue is constant, then how is the runtime complexity of the dequeue method O(n)?